hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastClientBeanDefinitionParser.java

```diff
@@ -228,11 +229,32 @@ private void handleSSLConfig(Node node, BeanDefinitionBuilder networkConfigBuild
                 String name = cleanNodeName(child);
                 if ("properties".equals(name)) {
                     handleProperties(child, sslConfigBuilder);
+                } else if ("host-verification".equals(name)) {
+                    handleHostVerification(child, sslConfigBuilder);
                 }
             }
             networkConfigBuilder.addPropertyValue("SSLConfig", sslConfigBuilder.getBeanDefinition());
         }
 
+        private void handleHostVerification(Node node, BeanDefinitionBuilder sslConfigBuilder) {
```

Comment:
Pull this method to `AbstractHazelcastBeanDefinitionParser` to avoid duplication in `HazelcastConfigBeanDefinitionParser`.

Reply:
:+1: